container/lxd/initialisation_test.go

```diff
@@ -336,7 +338,8 @@ func (s *InitialiserSuite) TestConfigureProxiesLXDNotRunning(c *gc.C) {
 	ctrl := gomock.NewController(c)
 	defer ctrl.Finish()
 	cSvr := lxdtesting.NewMockContainerServer(ctrl)
-	s.PatchForProxyUpdate(c, cSvr, false)
+	s.isLXDRunningLocally = false
```

Comment:
This won't work if `IsRunningLocally` is already patched in the setup, and `PatchForProxyUpdate` no longer patches that method.

Reply:
IsRunningLocally is patched with a function that returns the value s.isLXDRunningLocally. Setting it here changes it correctly.

Reply:
Right you are.